docs/SANITY_WEBHOOK_SETUP.md

```
@@ -0,0 +1,167 @@
+
+   **Filter** (optional, to track only specific documents):
+
+   ```groq
+   _type in ["news", "group", "person", "testimonial", "settings", "navigation"]
+   ```
+
+   **Projection** (what is sent to the webhook):
+
+   ```groq
+   {
+   	_type,
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 4

(MD010, no-hard-tabs)